demos/serverless/README.md

```diff
@@ -28,6 +28,19 @@ npm install
 npm run deploy -- -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting
 ```
 
+#### ChimeSDKMeetings Namespace vs Chime NameSpace
+The Chime and the ChimeSDKMeetings are both Amazon Chime's AWS clients to help builders create the Amazon Chime SDK meeting, attendees, enable transcription features and so on. Going forward, we will be using ChimeSDKMeetings client as the default client.
+
+For local development.....
+
+For serverless backend deployment, provide an additional parameter -u false if you want to use the Chime client over the ChimeSDKMeetings client.
```

Comment:
For local deployment, you can provide an additional parameter `-u false` if you want to use the legacy Chime SDK client over the ChimeSDKMeetings client.

Reply:
Lets combine these two sentences.